### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by umuthan**
UI Improvements:
* Line height fix for texts, now it is more readable,
* The entities visible at the bottom of the cards of each Beacon,
* The title and logos are in the conf file,
* An image for the Beacon's that has no logo,
* The example queries are under the input text with a toggle button.

### CODE REVIEW

File: static/sass/style.scss [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
+@import "reset.scss";
+
+/* @import "compass/reset"; */
+/* @import "compass/css3"; */
 
 @import "variables.scss";
-@import "main.scss";
\ No newline at end of file
+@import "main.scss";
+@import "admin.scss";
```
Comment (silverdaz) [inline static/sass/style.scss]: Remove it ;)

File: templates/card.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<!-- CARD -->
+<div class="card">
+    {% if remove_enabled %}
+      <button>
+        <i class="fas fa-trash"></i>
```
Comment (silverdaz) [inline templates/card.html]: no trash cuz no admin page

File: templates/card.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <img src="{{ service.logo_url }}" title="{{ service.name }} Logo" />
+        {% else %}
+        <img src="{{ static('img/no_logo.png') }}" title="Logo Unavailable" />
+        {% endif %}
+        <ul>
+          <li>
+            <a href="{{ service.visit_us }}"><i class="fas fa-globe"></i> Visit us</a>
+          </li>
+          <li>
+            <a href="{{ service.beacon_api }}"><i class="fas fa-share-square"></i>Beacon API</a>
```
Comment (silverdaz) [inline templates/card.html:14]: wrap them in if-then-else, in case we don't have the values

**Review by silverdaz — CHANGES_REQUESTED** [review]
Minor updates but overall good refactoring!

File: templates/card.html (hunk 2 above)
Comment (umuthan) [reply templates/card.html]: removed.

File: templates/card.html (hunk 3 above)
Comment (umuthan) [reply templates/card.html:14]: added the if statements each value

File: static/sass/style.scss (hunk 1 above)
Comment (umuthan) [reply static/sass/style.scss]: removed ;)

**Review by silverdaz — APPROVED** [review]
LGTM

**Pull request merged**